x/clerk/side_handler_test.go

```diff
@@ -0,0 +1,279 @@
+package clerk_test
+
+import (
+	// "fmt"
+	"math/big"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	tmprototypes "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	// ethTypes "github.com/maticnetwork/bor/core/types"
+	// common "github.com/maticnetwork/bor/common"
```

Comment:
let's remove these lines

Reply:
not removing as these will be used when some tests will be uncommented